modules/infra/api/gr_infra.h

```diff
@@ -53,6 +53,8 @@ struct gr_iface {
 
 // Info for GR_IFACE_TYPE_PORT interfaces
 struct gr_iface_info_port {
+#define GR_PORT_DRIVER_NAME_SIZE 16
```

Comment:
Can you check what is the max size for driver names in DPDK?

Reply:
done